scripts/snap-in-docker.sh

```diff
@@ -4,9 +4,14 @@
 
 set -ev
 
-docker run -v $GOPATH:/go snapcore/snapcraft sh -c 'cd /go/src/github.com/snapcore/snapweb && export GOPATH=/go PATH=/go/bin:$PATH && ./scripts/snap.sh'
+docker run -v $GOPATH:/go dbarth/snapweb-builder sh -c 'cd /go/src/github.com/snapcore/snapweb && export GOPATH=/go PATH=/go/bin:$PATH && ./scripts/snap.sh'
```

Comment:
This is to optimize the builder for snapweb, otherwise we download / install a ton of debs and npms at each run. See the Dockerfile.
The docker image is then hosted here : https://hub.docker.com/r/dbarth/snapweb-builder/